.eslintrc.yml

```diff
@@ -1,27 +1,61 @@
+extends: 
+  - "standard"
+  # - "plugin:requirejs/recommended"
+  # - "plugin:jsdoc/recommended"
+  # - "plugin:compat/recommended"
+
+parserOptions:
+  ecmaVersion: 2015
+
 env:
   es6: true
   browser: true
   amd: true
+  jquery: true
+
+plugins:
+  - requirejs
+  - deprecate
+  - jsdoc
+
+globals:
+  document: "readonly"
+  navigator: "readonly"
+  window: "readonly"
 
 rules:
-  block-spacing: ["error"]
-  brace-style: ["error"]
-  comma-dangle: ["error", "never"]
-  comma-spacing: ["error"]
-  eol-last: ["off"]
-  indent: ["error", 4, { "SwitchCase": 1 }]
-  keyword-spacing: ["error"]
-  line-comment-position: ["off"]
-  max-statements-per-line: ["error"]
-  no-empty: ["error"]
-  no-extra-semi: ["error"]
-  no-floating-decimal: ["error"]
-  no-multi-spaces: ["error"]
-  no-multiple-empty-lines: ["error", { "max": 1 }]
-  no-trailing-spaces: ["error"]
-  no-void: ["off"]
-  one-var: ["error", "never"]
-  padding-line-between-statements: ["off"]
-  semi: ["off"]
-  space-before-blocks: ["error"]
-  yoda: ["off"]
```

Comment:
I much prefer these manual rules until we have complete linting for the javascript, and even then I still prefer the manual rules to unknown rulesets. Why not just fix some of the minor ones for now rather than trying to validate 14,000 lines of changes in a single pull request?